plugins/techdocs/src/reader/components/TechDocsReaderPageContent/dom.tsx

```diff
@@ -512,31 +504,6 @@ export const useTechDocsReaderDom = (
                 font-size: var(--md-typeset-font-size);
               }
   
-              ${headings.reduce<string>((style, heading) => {
-                const styles = theme.typography[heading];
-                const { lineHeight, fontFamily, fontWeight, fontSize } = styles;
-                const calculate = (value: typeof fontSize) => {
-                  let factor: number | string = 1;
-                  if (typeof value === 'number') {
-                    // 60% of the size defined because it is too big
-                    factor = (value / 16) * 0.6;
-                  }
-                  if (typeof value === 'string') {
-                    factor = value.replace('rem', '');
-                  }
-                  return `calc(${factor} * var(--md-typeset-font-size))`;
-                };
-                return style.concat(`
-                  .md-typeset ${heading} {
-                    color: var(--md-default-fg-color);
-                    line-height: ${lineHeight};
-                    font-family: ${fontFamily};
-                    font-weight: ${fontWeight};
-                    font-size: ${calculate(fontSize)};
-                  }
-                `);
-              }, '')}
-  
```

Comment:
This was introduced to improve accessibility on documentation pages by changing the font-size to be calculated based on the base HTML font size. We can't remove these lines because today's font sizes are relative to the `--md-typeset-font-size` variable and this variable uses rem as the unit so that it is calculated based on the base HTML font size.

I would recommend keeping BackstageTheme as the source of truth for typography design tokens and with that keep consistency across all pages in a Backstage App (all pages using the same theme, all pages using the same typography design tokens) and only remove the 60% factor in headings font size calculation.

Replace line 522 by:
```
factor = (value / theme.typography.htmlFontSize ?? 16);
```
What do you think @emmaindal ?

Reply:
@camilaibs Thanks for providing context. I agree with you about using Backstage theme, ideally we do that for everything in TechDocs and not just some parts, that will improve the experience for everyone. I think the current issue with using the theme from Backstage right now is that not all headings (h5 & h6) have their fontSize defined. And `htmlFontSize` doesn't exist on `typography`, do you mean the specific headings `fontSize`? 

If so I would suggest either propose to add h5 and h6 to Backstage theme (then we can continue to use that). But please clarify what you refer to as `htmlFontSize` and I can update the PR.

Reply:
I mean typography `htmlFontSize` and also the individual headings font sizes that can be found in a Material UI theme, you can explore all tokens on this page:

https://v4.mui.com/customization/default-theme/#explore

<img width="875" alt="image" src="https://user-images.githubusercontent.com/6290749/168319385-dc73ac26-a549-4d8e-b43f-e986abb9a142.png">

Reply:
@emmaindal as we talked, I pushed the suggested changes, could you take a look?